src/backend/distributed/commands/table.c

```diff
@@ -178,6 +178,10 @@ PreprocessDropTableStmt(Node *node, const char *queryString,
  *
  * This function also processes CREATE TABLE ... PARTITION OF statements via
  * PostprocessCreateTableStmtPartitionOf function.
+ *
+ * Also CREATE TABLE ... INHERITS ... commands are filtered here. If the inherited
```

Comment:
Very Minor:

I would write something like the following:

```
This function processes `CREATE TABLE ... INHERITS ...`, and errors out if a local table inherits a distributed table
```

but current version is absolutely fine as well.